src/rsvc/srv.c

```diff
@@ -1153,7 +1153,7 @@ bcast_create(crt_opcode_t opc, bool filter_invert, d_rank_list_t *filter_ranks,
 				    NULL /* co_bulk_hdl */, NULL /* priv */,
 				    filter_invert ?
 				    CRT_RPC_FLAG_FILTER_INVERT : 0,
-				    crt_tree_topo(CRT_TREE_FLAT, 0), rpc);
+				    crt_tree_topo(CRT_TREE_KNOMIAL, 0), rpc);
```

Comment:
Could you please explain this change?  And the branch ratio has to be [2-64] see comments above crt_tree_topo().

 * \param[in] branch_ratio     branch ratio, be ignored for CRT_TREE_FLAT.
 *                             for KNOMIAL tree or KARY tree, the valid value
 *                             should within the range of
 *                             [CRT_TREE_MIN_RATIO, CRT_TREE_MAX_RATIO], or
 *                             will be treated as invalid parameter.